patch/cisco-npu-disable-other-bars.patch

```diff
@@ -0,0 +1,81 @@
+From 0be339b4c8468138dfe1a30de29511e94b846af6 Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Mon, 16 Aug 2021 12:55:37 -0700
+Subject: [PATCH] NPU disable unused PCI BAR
```

Comment:
Please make it an upstreamable commit message summary ([How to Write a Git Commit Message][1]).

[1]: https://chris.beams.io/posts/git-commit/